yt/utilities/lib/pixelization_routines.pyx

```diff
@@ -1128,6 +1136,12 @@ def interpolate_sph_grid_gather(np.float64_t[:, :, :] buff,
         np.float64_t[:] quantity_to_smooth, PyKDTree kdtree,
         int use_normalization=1, kernel_name="cubic", pbar=None,
         int num_neigh=32):
+    '''
```

Comment:
super minor but docstrings should be delimited by triple double quotes, not single quotes